doc/swish/http.tex

```diff
@@ -533,7 +536,7 @@ \subsection{WebSocket Protocol}\label{sec:websocket-protocol}
 
 \defineentry{http:respond}
 \begin{procedure}
-  \code{(http:respond \var{conn} \var{status} \var{header-alist} \opt{\var{content}})}
+  \code{(http:respond \var{conn} \var{status} \var{header-alist} \opt{\var{content}} \opt{\var{timeout}})}
```

Comment:
When timeout is specified, content is required, so you might use `\opt{\var{content} \opt{\var{timeout}}}` instead.

Reply:
That's a nice change. Thanks.